dataprofiler/tests/profilers/test_boolean_option.py

```diff
@@ -0,0 +1,90 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BooleanOption
+from dataprofiler.tests.profilers.test_base_option import TestBaseOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBooleanOption(TestBaseOption):
+	
+	@classmethod
+	def setUpClass(cls):
+		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
+	
+	@classmethod
+	def getOptions(self, **params):
+		return BooleanOption(**params)
+	
+	@classmethod
+	def getOptionsPath(self, **params):
+		return "BooleanOption"
```

Comment:
This could be a class variable instead of a method. I wasn't sure if the params of the Option classes may eventually change the variable path.